pkg/controller/plan/kubevirt.go

```diff
@@ -2276,19 +2276,20 @@ func (r *KubeVirt) CreatePvcForNfs(pvcNamePrefix string, pvName string) (pvcName
 		Name:      pvc.Name,
 	}
 
-	if err = wait.PollUntilContextTimeout(context.TODO(), 5*time.Second, 45*time.Second, true, func(ctx context.Context) (done bool, err error) {
-		err = r.Get(context.TODO(), pvcNamespacedName, pvc)
-		if err != nil {
-			r.Log.Error(err, "Failed to get OVA plan PVC")
-			return false, err
-		}
-		return pvc.Status.Phase == core.ClaimBound, nil
-
-	}); err != nil {
-		r.Log.Error(err, "Failed to bind OVA PVC to PV ")
-		return
+	go func() {
```

Comment:
I think this doesn't actually do anything, I mean, it checks for the status of the PVC asynchronously, but nothing is changed based on the result

Reply:
you right, but I saw that I don't get any errors with that implementation and then I realize that the current code is redundant since the virt-v2v pod knows how to handle this behavior by itself.

I dink the best solution will be to remove entirely this check from the code but the downside will be we don't have the error in the case the pod stack on pending....